elements/pfe-navigation/demo/index.html

```diff
@@ -250,9 +254,206 @@ <h3>
 
     <div style="height: 200px"></div>
 
-    <h2>Large Logo</h2>
+    <h2>Small Logo with Extra Content in Mega Menu Tray and Demo of Single Column Dropdown</h2>
+    <pfe-navigation id="pfe-navigation">
+      <!-- todo: figure out to do the overlay -->
+      <div class="pfe-navigation__overlay"></div>
+
+      <nav class="pfe-navigation" aria-label="Main Navigation">
```

Comment:
I'm not seeing the nested components necessary like pfe-navigation-main and pfe-navigation-item. There are also a few items from the shadow template in demo file. @chrisdo1 can help with any questions too.

Reply:
We're trying a very different approach for the universal Red Hat nav, trying to avoid too many sub components.